doc/misc/run_bisq_via_vps.md

```diff
@@ -0,0 +1,63 @@
+## Hosted Bisq using VPS
+
+
+From: https://github.com/bisq-network/exchange/issues/511
+
+It's possible to install and view Bisq using a Google Compute Engine instance. This is handy and allows offers to always be visible, as well as the possibility to inspect the status of the client.
+
+This is how to host Bisq:
+
+Go to Google compute engine (https://cloud.google.com/compute/) and start an instance using the free trial. One gets $300 credit and 60 days trial period. I used one CPU, 40 GByte of boot disk, and the default value of RAM, 3.75 GB.
+I chose Ubuntu 16.04 LTS as the operating system.
+
+How to make a project and start an instance is explained in: https://goo.gl/1Ljy7O which also explains how to install a vnc-server. In short, ssh to your instance, which can be done from the google compute engine (gce) console, then execute the following script:
```

Comment:
In short, **add** ssh to

Reply:
Google should be uppercase (being picky now ;-))

Reply:
> In short, add ssh to
I have not written that but I think it was meant to connect via ssh